src/test/java/hudson/plugins/sshslaves/SSHLauncherTest.java

```diff
@@ -74,6 +77,8 @@
 
 public class SSHLauncherTest {
 
+    private static final VersionNumber JAVA_8_MINIMAL_SINCE = new VersionNumber("2.54");
```

Comment:
Please avoid copy-pasting from https://github.com/jenkinsci/ssh-slaves-plugin/blob/f0634f15bdc10a67cc364a191875dc258d127334/src/main/java/hudson/plugins/sshslaves/JavaProvider.java#L45-L47 . If you need it, it is better to extend the JavaProvider API if needed. 🐛